src/scripts/destinyTrackerApi/reviewDataCache.js

```diff
@@ -116,6 +118,34 @@ class ReviewDataCache {
   }
 
   /**
+   * Should we consider this item locally cached with changes?
+   *
+   * @param {any} item
+   * @param {bool} isCached
+   * @memberof ReviewDataCache
+   */
+  _markItemAsLocallyCached(item,
+                           isCached) {
+    item.isLocallyCached = isCached;
+  }
+
+  markItemAsReviewedAndSubmitted(item,
+                                 userReview) {
+    this._markItemAsLocallyCached(item, false);
+    const matchingItem = this._getMatchingItem(item);
+
+    if (matchingItem.reviews) {
+      matchingItem.reviews = _.reject(matchingItem.reviews, { isReviewer: true });
+    }
+
+    if (!matchingItem.reviews) {
```

Comment:
Oh yeah.  That was a holdover from a mistake I was making earlier using the wrong underscore function.  Will fix.